PLAYER/StartPlayerTeleport.md

```diff
@@ -17,3 +25,18 @@ void START_PLAYER_TELEPORT(Player player, float x, float y, float z, float headi
 * **teleportWithVehicle**: Teleports the player along with the vehicle they are in.
 * **findCollisionLand**: Attempt to find a ground coordinate at the given XY location; overriding the Z value.
 * **p7**: 
+
+## Examples
+
+```lua
+local coords = vector3(100.0, 100.0, 50.0)
+StartPlayerTeleport(PlayerId(), coords.x, coords.y, coords.z, 0.0, false, true, true)
+
+while IsPlayerTeleportActive() do
+  i = i + 1
```

Comment:
`i = i + 1` ?
it's not even used anywhere, why?

Reply:
Accidently left this in here from a code snippet I used to try to figure out the amount of frames a teleport would take, I removed it in the new commit.